### DISCUSSION TITLE
Browser Extension?

### DISCUSSION THREAD

**Discussion by secasa79** [FlareSolverr/FlareSolverr#348] [Ideas]
I'm new to this and am having trouble understanding how to use FlareSolverr since I am not code-savvy. I was wondering if it would be possible for FlareSolverr to be utilized as a web browser extension if one was created? If not, then why?

**Comment by ngosang** [comment]
It can not be used as a browser extension. It's not planned to write a browser extension but it should be possible to write one extension with similar features/goals.
